ResearchKit/Common/ORKTask.h

```diff
@@ -214,6 +214,11 @@ requests access to these HealthKit types.
  */
 @property (nonatomic, readonly) BOOL providesBackgroundAudioPrompts;
 
+/**
+ *  Validates task parameters. Will throw excpetion if validation fails.
```

Comment:
Comment formatting - we don't have the prefix star. After the initial sentence, we insert two newlines and start a new paragraph to give additional context.

Might also be nice to describe what types of validation are expected here, and whether you're expected to validate your steps as well (no; because that's done later, while going through the steps; the task is not expected to know all its steps in advance).